mx.jruby/mx_jruby.py

```diff
@@ -0,0 +1,58 @@
+import mx
+import os
+from os.path import join, exists
+import subprocess
+
+_suite = mx.suite('jruby')
+
+
+def build(args):
+    mx.log('Building JRuby')
+
+    rubyDir = _suite.dir
+
+    # HACK: since the maven executable plugin does not configure the
+    # java executable that is used we unfortunately need to append it to the PATH
+    javaHome = os.getenv('JAVA_HOME')
+    if javaHome:
+        os.environ["PATH"] = os.environ["JAVA_HOME"] + '/bin' + os.pathsep + os.environ["PATH"]
+
+    mx.logv('Setting PATH to {}'.format(os.environ["PATH"]))
+    mx.logv('Calling java -version')
+    mx.run(['java', '-version'])
+
+    def apply_to_file(filename, function):
+        contents = open(filename).read()
+        contents = function(contents)
+        open(filename, 'w').write(contents)
+
+#    # Repository and commit of JRuby we want to build
+#    commit = open(_rubyVersionFile()).read().strip()
+
+    # Truffle version
+
+    truffle = mx.suite('truffle')
+    truffle_commit = subprocess.check_output(
```

Comment:
This should be done through the `VC` abstractions from mx (i.e., `truffle_commit = truffle.vc.parent(truffle.dir)`)